ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1688,20 +1691,17 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
             width: frame.width - separatorLeadingInset(for: type),
             height: separator.frame.height
         )
+        separator.flipForRTL()
     }
 
     func separatorLeadingInset(for type: SeparatorType) -> CGFloat {
-        switch type {
-        case .none:
+        guard type == .inset else {
```

Comment:
Question: this logic differs between `main` and `fluent2-tokens`. Which is more correct? Which should we go with in the future?

Reply:
@sophialee0416

Reply:
In the future, we should go with the switch statement. A little context: The SwiftUI Divider control takes safe insets into consideration, which was causing a regression with the divider appearance (because MSFSeparator did not). So to summarize, switch statement + MSFDivider, guard + MSFSeparator
https://github.com/microsoft/fluentui-apple/pull/1167

Reply:
Great, thanks!